companies.py

```diff
@@ -0,0 +1,4 @@
+eleks_company = ['Toni', 'Anna', 'Bob', 'Den']
+toshiba_company = ['Marry', 'Tod', 'Den', 'Anna']
+toshiba_company.extend(eleks_company)
+print(*toshiba_company, sep=', ')
```

Comment:
not fully completed. after your manipulations, Eleks should not have employees. So please add one line of code with fix

Reply:
was fixed. Done